CO2_Gadget_EINK.h

```diff
@@ -17,8 +17,10 @@
 // clang-format on
 #include <GxEPD2_BW.h>
 
-uint16_t redrawDisplayEveryCycles = 10;  // Redraw display every X partial updates
-uint16_t cyclesLeftToRedrawDisplay = 0;  // Cycles left to redraw display
+// Variables for e-ink functionality -> move to platformio.ini or CO2_Gadget.ino in a future for menu config
+uint16_t co2_HYSTERESIS = 10;           // Minimum change in CO2 value to refresh display
+int16_t timeBetweenEinkUpdates = 10;    // 10 seconds between refresh display
```

Comment:
I can't remember but maybe this is already defined (awaiting the low-power-eink merge). 
If not, this should be implemented as   `deepSleepData.redrawDisplayEveryCycles`

Reply:
I think this variable is needed in e-Ink even when not low power is not implemented. Is something needed by all E-ink displays.